src/components/StorageOverview/Utilization/Utilization.js

```diff
@@ -55,13 +61,28 @@ export const Utilization = ({
   if (latencyStats) {
     latencyStatsMax = Math.max(0, ...latencyStats);
   }
+  const { timestamps } = throughputData;
 
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>Performance</DashboardCardTitle>
+        <DashboardCardTitle>Utilization</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
+        <Row>
+          <Col className="kubevirt-utilization__time-duration kubevirt-utilization__time-duration--left">Time </Col>
+          <Col className="kubevirt-utilization__time-duration kubevirt-utilization__time-duration--right">
+            6 hour average
+          </Col>
+        </Row>
+        <ChartAxis
+          scale={{ x: 'time' }}
+          tickValues={timestamps}
+          tickFormat={x => formatToShortTime(x)}
```

Comment:
The `x` can be potentially `null`.

Reply:
The Axis defaults to the times `12am, 6am, 12pm` and so on when the value of x is `null`